### PR TITLE
Add `$ETCD_DATA_DIR` to system environment

### PR DESCRIPTION
**Pull request by stevekuznetsov**
We need `etcd` to store it's data in a tmpfs to ensure that the weird
semantics around IO on EC2 EBS don't cause `etcd` to fall over in the
middle of our test suites. We don't control the installation of `etcd`
so we try to influence storage via system-wide environment.

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/cc @smarterclayton @liggitt does this seem reasonable?

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
fixes https://github.com/openshift/aos-cd-jobs/pull/159

File: config/common/test_cases/origin.yml [hunk 1]
```diff
@@ -10,9 +10,11 @@ actions:
   - type: "script"
     title: "use a ramdisk for etcd"
     script: |-
-      sudo mkdir -p /tmp
-      sudo mount -t tmpfs -o size=2048m tmpfs /tmp
-      sudo restorecon -R /tmp
+      sudo su root
+      mkdir -p /tmp/etcd
```
Comment (liggitt) [inline config/common/test_cases/origin.yml]: why the etcd subdir? doesn't this get shadowed by the mount?

Comment (stevekuznetsov) [inline config/common/test_cases/origin.yml]: Good point

**Comment by stevekuznetsov** [comment]
@smarterclayton are you OK with this approach?

**Pull request merged**